test/common/TestModels.scala

```diff
@@ -433,4 +433,64 @@ object TestModels {
   val whatYouOweUrl: String = testOnly.controllers.routes.WhatYouOweController.show.url
 
   val vatDetailsUrl: String = controllers.routes.VatDetailsController.details.url
+
+  val nonInterestChargeTypes: Seq[ChargeType] = Vector(
```

Comment:
To make this more concise and maintainable, can you:

- Change the interest & penalty charge type lists in `ChargeType` to be Sets of type `Set[ChargeType]` (should have been this anyway tbh to match `allChargeTypes`)
- Update the val here to simply be `allChargeTypes.diff(interestChargeTypes)` (or preferably if this is only being used in one unit test, move it there)